### PR TITLE
Update using-printstream-format-to-format-a-string-before-printing.md

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Topic

| Topic Name | Error Message |
|:-:|:-:|
| error-not-found | Topic should have at least one workout with the "workout" aspect. |
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

File: java/data-management/strings-ii/using-printstream-format-to-format-a-string-before-printing.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ links:
 int i = 500;
 float f = 2.36;
 String s = "Java";
-//System.out is Prtinstream object
+//System.out is Printstream object
```
Comment (kapnobatai137) [inline java/data-management/strings-ii/using-printstream-format-to-format-a-string-before-printing.md:24]: we should add a space after comment characters (`//`) and I think the name is `PrintStream` ([source](https://docs.oracle.com/javase/7/docs/api/java/io/PrintStream.html))

```suggestion
// System.out is a PrintStream object
```

**Review by kapnobatai137 — CHANGES_REQUESTED** [review]

**Comment by enki-hq** [comment]
### Topic

| Topic Name | Error Message |
|:-:|:-:|
| error-not-found | Topic should have at least one workout with the "workout" aspect. |
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Topic

| Topic Name | Error Message |
|:-:|:-:|
| error-not-found | Topic should have at least one workout with the "workout" aspect. |
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Topic

| Topic Name | Error Message |
|:-:|:-:|
| error-not-found | Topic should have at least one workout with the "workout" aspect. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Pull request merged**